src/webui/www/private/views/transferlist.html

```diff
@@ -106,6 +109,14 @@
             offsets: {
                 x: -15,
                 y: 2
+            },
+            onShow: function() {
+                // Hides rename, and renameFiles menu items when multiple torrents are selected
```

Comment:
I don't know if this is the right place to do this but other items in the context menu are hidden in the function below
(this line points on how to check for metadata)
`if (data['state'] == 'metaDL' || data['state'] == 'forcedMetaDL')`
https://github.com/qbittorrent/qBittorrent/blob/915121a0dd2cedad2912eddea194c53bf4b5500c/src/webui/www/private/scripts/contextmenu.js#L335

Reply:
thanks! i'll get this added

Reply:
i added a change to only allow renaming when 1 item is selected and also when metadata is `not` downloaded

https://github.com/loligans/qBittorrent/blob/f563267f0bac867d008f06fb81aabf67c19c0b23/src/webui/www/private/scripts/contextmenu.js#L365-L372

Reply:
my mistake, i didn't know of a good way to create a torrent that hadn't downloaded metadata. i figured out how and got it squared away. on a side note, i only hide the `renameFiles` menu item now because `rename` can be done on a torrent that is still downloading metadata which i was unaware of. i don't plan on touching anything else for the rest of this PR